src/vmm/src/device_manager/mmio.rs

```diff
@@ -435,6 +479,12 @@ impl MMIODeviceManager {
     }
 }
 
+impl Aml for MMIODeviceManager {
```

Comment:
We need _some_ method to give access to DSDT data of `MMIODeviceManager`.

Reply:
Why not just access `dsdt_data` directly as it is `pub`?
```rust 
// this
dsdt_data.extend_from_slice(&mmio_device_manager.dsdt_data);

// instead of this
mmio_device_manager.append_aml_bytes(&mut dsdt_data);
```

Reply:
Can do that!

Reply:
I think it is.

Reply:
Again, I'm fine to do the change, but it's really useful for the reviews if you explain "why" you think this is important.

Reply:
The reasons are:
- This trait is not used as a trait bound anywhere
- We can access `dsdt_data` directly, without need for any traits